combine/bygroup.go

```diff
@@ -0,0 +1,24 @@
+package combine
+
+import (
+	"io"
+	"os"
+)
+
+const numSummaryColumns = 2
+const groupSumCols = 2
+
+func MergeByGroupFiles(inputs []*os.File, output *os.File) error {
+	return Merge(inputs, output, mergeGroupStreamToFile)
+}
+
+// mergeGroupStreamToFile merges pre-sorted (pre-merged) group data
+// (eg. from a `sort -m` of .bygroup files), summing consecutive lines with
+// the same first 2 columns, and outputting the results.
+func mergeGroupStreamToFile(data io.ReadCloser, output io.Writer) error {
```

Comment:
No need for this function. Just use the MergeSummaryLines call directly.

Reply:
The function still exists. You can delete this whole function and call MergeSummaryLines inside MergeByGroupFiles().